include/common/s1ap_structs.h

```diff
@@ -436,6 +446,8 @@ typedef struct proto_IE_data {
         struct eRAB_elements 	erab;
         ue_aggregate_maximum_bitrate ue_aggrt_max_bit_rate;
         ERABSetup E_RABToBeSetupItemCtxtSUReq;
+	struct ERABSetupList eRABSetupList;
```

Comment:
Clearly no consistency in variable names... ue_aggregate_maximum_bitrate   , ERABSetup (typedefed).... struct ERABSetupList.... Please see if we can keep it consistent.

Reply:
Done.